com.unity.render-pipelines.high-definition/Editor/RayTracing/PlanarReflectionAngleGenerator.cs

```diff
@@ -0,0 +1,143 @@
+using System.IO;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    internal class PlanarReflectionAngleGenerator : MonoBehaviour
```

Comment:
I may have missed something but I did not see where this class was actually used?
Also, why is it a MonoBehaviour? It looks like it could just be a helper class with static functions. Having a MonoBehaviour just for that seems costly for no reasons.